opm/simulators/aquifers/BlackoilAquiferModel_impl.hpp

```diff
@@ -158,20 +158,16 @@ BlackoilAquiferModel<TypeTag>::init()
         const AquiferCT aquiferct = AquiferCT(eclState.getTableManager(), deck);
         const Aquancon aquifer_connect = Aquancon(eclState.getInputGrid(), deck);
 
-        std::vector<AquiferCT::AQUCT_data> aquifersData = aquiferct.data();
-        std::vector<Aquancon::AquanconOutput> aquifer_connection = aquifer_connect.getAquOutput();
-
         assert(aquifersData.size() == aquifer_connection.size());
```

Comment:
This assertion will not compile without `-DNDEBUG`.

Reply:
fixed